domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -544,60 +536,39 @@ class TopicListController @Inject constructor(
   }
 
   // TODO(#2550): Remove hardcoded order of topics. Compute list of suggested stories from backend structures
-  /** Returns a list of topic IDs for which the specified topic ID expects to be completed before being suggested.
-   * Dependent hierarchy
-   * TEST_TOPIC_ID_0 (Fractions)
-   * TEST_TOPIC_ID_1 (TEST_TOPIC_ID_0,Ratios)
-   * Addition and Subtraction (depends on Place Values)
-   * Multiplication (depends on Addition and Subtraction)
-   * Division (depends on Multiplication)
-   * Expressions and Equations (depends on A+S, Multiplication, Division)
-   * Fractions (depends on A+S, Multiplication, Division)
-   * Ratios (depends on A+S, Multiplication, Division)
-   * Decimals (depends on A+S, Multiplication, Division)
-   *
+  /** Returns a list of topic IDs for which the specified topic ID expects to be completed before
+   * being suggested.
    * */
   private fun retrieveTopicDependencies(topicId: String): List<String> {
```

Comment:
Please follow the KDoc style guide: e.g.:

```kotlin
/**
 * ...
 */
```

Reply:
Done.